app/controllers/jobs_controller.rb

```diff
@@ -47,6 +47,6 @@ def add_job
   # TODO: Remove #create and #job_params?
 
   def job_params
```

Comment:
job_params can be deleted, there's no create method that calls it

Reply:
Ok will do